mail_tracking/models/mail_tracking_email.py

```diff
@@ -473,3 +473,25 @@ def event_process(self, request, post, metadata, event_type=None):
         # - return 'NONE' if this request is not for you
         # - return 'ERROR' if any error
         return "NONE"  # pragma: no cover
+
+    @api.model
+    def _gc_mail_tracking_email(self, max_age_days=180, limit=5000):
+        enable_deletion = (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("mail_tracking.enable_mail_tracking_email_deletion_job")
+        )
+
+        if not enable_deletion:
+            _logger.info("Mail tracking email deletion job is disabled.")
+            return False
+
+        target_write_date = fields.Datetime.subtract(
+            fields.Datetime.now(), days=max_age_days
+        )
+        domain = [
+            ("write_date", "<", target_write_date),
+        ]
```

Comment:
You should extract the domain computation in a separate method, so it's easy to override to fine-tune the domain (if we want to keep failure values for instance).

Reply:
Done